test/error.js

```diff
@@ -91,59 +87,55 @@ test('failed is true on failure', async t => {
 	t.true(failed);
 });
 
-test('error.killed is true if process was killed directly', async t => {
+test('error.isTerminated is true if process was killed directly', async t => {
 	const subprocess = execa('forever.js');
 
 	subprocess.kill();
 
-	const {killed} = await t.throwsAsync(subprocess, {message: /was killed with SIGTERM/});
-	t.true(killed);
+	const {isTerminated} = await t.throwsAsync(subprocess, {message: /was killed with SIGTERM/});
+	t.true(isTerminated);
 });
 
-test('error.killed is false if process was killed indirectly', async t => {
+test('error.isTerminated is true if process was killed indirectly', async t => {
 	const subprocess = execa('forever.js');
 
 	process.kill(subprocess.pid, 'SIGINT');
 
 	// `process.kill()` is emulated by Node.js on Windows
 	const message = process.platform === 'win32' ? /failed with exit code 1/ : /was killed with SIGINT/;
-	const {killed} = await t.throwsAsync(subprocess, {message});
-	t.false(killed);
+	const {isTerminated} = await t.throwsAsync(subprocess, {message});
+	t.true(isTerminated);
```

Comment:
Change of behavior: now `true` regardless of whether the process was terminated directly or not.